server/api/core/core.js

```diff
@@ -203,10 +206,22 @@ export default {
     return settings.settings || {};
   },
 
+  getShopCurrency() {
```

Comment:
Did you intend to add this function twice? If so, see my comment on other instance.